app/code/Magento/Sales/Test/Unit/Model/Order/InvoiceTest.php

```diff
@@ -156,17 +160,13 @@ public function testGetOrderLoadedById()
     {
         $orderId = 100000041;
         $this->modelWithoutOrder->setOrderId($orderId);
-        $this->order->expects($this->once())
-            ->method('load')
-            ->with($orderId)
-            ->willReturnSelf();
+        $this->orderRepository->expects($this->once())
```

Comment:
It may be a good idea to check both cases: 
* in-object cache is used
* new entity is created